test/nomad_helpers.go

```diff
@@ -233,3 +291,61 @@ func callNomadApi(t *testing.T, nodeIpAddress string, path string) ([]interface{
 
 	return result, nil
 }
+
+func runNomadClusterSSHTest(t *testing.T, packerBuildName string, ssh_username string) {
+	examplesDir := test_structure.CopyTerraformFolderToTemp(t, REPO_ROOT, "/")
+
+	defer test_structure.RunTestStage(t, "teardown", func() {
+		terraformOptions := test_structure.LoadTerraformOptions(t, examplesDir)
+		terraform.Destroy(t, terraformOptions)
+
+		amiId := test_structure.LoadAmiId(t, examplesDir)
+		awsRegion := test_structure.LoadString(t, examplesDir, SAVED_AWS_REGION)
+		aws.DeleteAmi(t, awsRegion, amiId)
+	})
+
+		test_structure.RunTestStage(t, "setup_ami", func() {
```

Comment:
Could you run `go fmt` on this code? The indentation looks off.

Reply:
Ah, I see - thanks for pointing this out. It should be fixed now.